### PR TITLE
resin-fsl-arm: add support for NXP MCIMX6Q-SDB board

### PR DESCRIPTION
**Pull request by ajlennon**
Signed-off-by: Alex J Lennon ajlennon@dynamicdevices.co.uk

### CODE REVIEW

File: mcimx6q-sdb.coffee [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		osx: 'http://docs.resin.io/#/pages/installing/gettingStarted-mcimx6q-sdb.md#on-mac-and-linux'
+		linux: 'http://docs.resin.io/#/pages/installing/gettingStarted-mcimx6q-sdb.md#on-mac-and-linux'
+	supportsBlink: false
+
+	yocto:
+		machine: 'imx6qsabresd'
+		image: 'resin-image'
+		fstype: 'resin-sdcard'
+		version: 'yocto-jethro'
+		deployArtifact: 'resin-image-mcimx6q-sdb.resin-sdcard'
```
Comment (agherzan) [inline mcimx6q-sdb.coffee]: Wasn't this supposed to be resin-image-MACHINE.resin-sdcard? Might be a typo.

File: mcimx6q-sdb.coffee [hunk 2]
```diff
@@ -0,0 +1,34 @@
+deviceTypesCommon = require 'resin-device-types/common'
```
Comment (agherzan) [inline mcimx6q-sdb.coffee]: By convention the filename is always <YOCTO_MACHINE>.coffee . This is necessary for barys to find the right json file easily.

File: mcimx6q-sdb.coffee (hunk 1 above)
Comment (floion) [inline mcimx6q-sdb.coffee]: Hi @ajlennon , can you redo the pull request as suggested by Andrei?

File: mcimx6q-sdb.coffee [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	slug: 'mcimx6q-sdb'
+	aliases: [ 'mcimx6q-sdb' ]
+	name: 'mcimx6q-sdb'
+	arch: 'armv7hf'
+	state: 'released'
```
Comment (floion) [inline mcimx6q-sdb.coffee]: maybe have this state "experimental" until some tests are conducted on this board

File: mcimx6q-sdb.coffee (hunk 1 above)
Comment (floion) [inline mcimx6q-sdb.coffee]: Hi Alex, can you follow up with the changes Andrei suggested?

File: mcimx6q-sdb.coffee (hunk 2 above)
Comment (floion) [inline mcimx6q-sdb.coffee]: Hi @ajlennon , can you follow up with the changes Andrei suggested?

The .coffee name should be <yocto-machine-name>.coffee

Comment (ajlennon) [inline mcimx6q-sdb.coffee]: @agherzan @floion hadn't seen this. Sure I'll make the change

File: mcimx6q-sdb.coffee (hunk 3 above)
Comment (agherzan) [inline mcimx6q-sdb.coffee]: Definitely this is an important change too.

**Comment by agherzan** [comment]
@ajlennon Kindly pinging you for this.

**Comment by floion** [comment]
@ajlennon are you still interested in getting this merged?

**Comment by agherzan** [comment]
Let's close this PR for now. As far as I know Alex is not working on this anymore. We can pick it up in the future if needed.

**Pull request closed without merging**